docs/delegate.md

```diff
@@ -1,14 +1,13 @@
 # Delegation
 
-For a more flexible configuration, you can have `server_name`
-resources (eg: `@user:example.com`) served by a different host and
-port (eg: `synapse.example.com:443`).
+Without configuring delegation, homeservers will expect the server
+responsible for resources using e.g. `example.com` as their `server_name`
+(e.g. `@user:example.com`) to be served at `example.com:8448`.
 
-Without configuring delegation, the matrix federation will
-expect to find your server via `example.com:8448`. The following methods
-allow you retain a `server_name` of `example.com` so that your user IDs, room
-aliases, etc continue to look like `*:example.com`, whilst having your
-federation traffic routed to a different server (e.g. `synapse.example.com`).
+Delegation is a Matrix feature allowing a homeserver admin to retain a
+`server_name` of `example.com` so that your user IDs, room aliases, etc
```

Comment:
you'd need to say *their* user IDs to match the third-person subject "a homeserver admin", but tbh it probably reads better if you just miss out the possessive:

```suggestion
`server_name` of `example.com` so that user IDs, room aliases, etc
```